src/wp-admin/about.php

```diff
@@ -61,9 +61,9 @@
 		</div>
 
 		<div class="about__section has-2-columns">
-			<div class="column">
+			<div class="column is-vertically-aligned-center">
 				<div class="about__image">
-					<img src="data:image/svg+xml,%3Csvg width='436' height='436' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%23dcdcde' /%3E%3C/svg%3E" alt="" />
+					<img src="https://s.w.org/images/core/6.3/1-site-editor.webp" alt="" />
```

Comment:
Why we use `.webp` images? Previously we use `.png` images for about page.

Reply:
[We used `.webp` in 6.1](https://core.trac.wordpress.org/browser/branches/6.1/src/wp-admin/about.php#L142), I believe the switch back to `png` last release was not intentional. The webp files are on average half the size of optimized pngs.